itou/users/models.py

```diff
@@ -303,6 +303,10 @@ def can_add_nir(self, job_seeker):
     def is_created_by(self, user):
         return bool(self.created_by_id and self.created_by_id == user.pk)
 
+    @property
+    def is_created_by_sso(self):
```

Comment:
Je ne suis pas convaincue par le nom de cette propriété. Dans le cas d'Inclusion Connect, les anciens comptes ne seront pas vraiment _créés_ par un SSO mais plutôt liés à un moment.

Voici quelques autres idées (par ordre de préférence) :
- has_sso_provider : c'est peut-être le plus explicite.
- has_used_sso
- is_bound_to_sso : c'est moche
- is_sso_authenticated : laisse penser qu'on teste l'authentification alors qu'on vient juste récupérer le fournisseur.

Reply:
`is_local_user` à utiliser en `not is_local_user` ?

Reply:
Voire `is_local` tout court avec `not user.is_local`

Reply:
Hum... :thinking: 
`is_local` ne me parle pas trop car je trouve ça un peu trop vaste. Je comprends l'idée dans le contexte mais je ne suis pas sure que je trouverais ça évident en le lisant hors contexte.
Si tu as 5 minutes, on peut y réfléchir ensemble.

Reply:
Je viens de relire ta première phrase et j'avoue que `is_local` ne fonctionne pas dans ce contexte, mea culpa. J'avais trop lié ma réflexion au contenu actuel du code de la fonction en question, sans repenser à l'objectif futur.
Dans ce cas je pense que `has_sso_provider` sera en effet plus future-proof.

Reply:
Merci ! J'ai donc modifié la propriété.